backend/backend/settings.py

```diff
@@ -27,7 +27,7 @@
 SECRET_KEY = 'django-insecure-mz0gymvj@n5wl2p0yau(vj0e3jdx_wok78+ead*=p4)$w)g5(z'  # TODO generate new one and keep secret
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False

```

Comment:
voor de zekerheid, dit best niet pushen.
Btw kunnen we dit mss niet best in de .env zetten? dat het locally true is en op de server false